doc/user/spark.md

```diff
@@ -14,8 +14,8 @@ to access DAOS from Hadoop and Spark.
 
 1. Linux OS
 2. Java 8
-3. Hadoop 2.7.6
-4. Spark 3
+3. Hadoop 3.2 or later
+4. Spark 3.3 or later
```

Comment:
3.3? Can this still work with Spark 3.0?

Reply:
it's typo. I meant spark 3.1. And spark-3.1.1 starts using hadoop 3.2 as default hadoop version. I am going to test with spark 3.1.1 and hadoop 3.2.0. So what's your suggestion?

Reply:
change it to 3.0